mysql-test/suite/tokudb/r/fast_update_binlog_mixed.result

```diff
@@ -1,17 +1,25 @@
 include/master-slave.inc
+Warnings:
+Note	####	Sending passwords in plain text without SSL/TLS is extremely insecure.
+Note	####	Storing MySQL user name or password information in the master info repository is not secure and is therefore not recommended. Please consider using the USER and PASSWORD connection options for START SLAVE; see the 'START SLAVE Syntax' in the MySQL Manual for more information.
 [connection master]
 set default_storage_engine='tokudb';
 create table tt (id int primary key, x int);
-set session tokudb_disable_slow_upsert=1;
+set session sql_mode=(select replace(@@sql_mode,'STRICT_TRANS_TABLES',''));
+set session sql_mode=(select replace(@@sql_mode,'STRICT_ALL_TABLES',''));
+set session tokudb_enable_fast_update=1;
 show variables like 'binlog_format';
 Variable_name	Value
 binlog_format	MIXED
 show variables like 'server_id';
 Variable_name	Value
 server_id	1
+set session sql_mode=(select replace(@@sql_mode,'STRICT_TRANS_TABLES',''));
+set session sql_mode=(select replace(@@sql_mode,'STRICT_ALL_TABLES',''));
+set session tokudb_enable_fast_update=1;
 show variables like 'binlog_format';
 Variable_name	Value
-binlog_format	MIXED
+binlog_format	STATEMENT
```

Comment:
Disagrees with the testcase name

Reply:
Those lines are for the slave, not for the master, so the test name corresponds to what the test does. But those checks are, actually, needless, because all necessary checks are performed in 'have_binlog_format*.inc" files. So I have just removed it.